spl/src/token_interface.rs

```diff
@@ -75,3 +75,6 @@ impl anchor_lang::Ids for TokenInterface {
 }
 
 pub use crate::token_2022::*;
+
+#[cfg(feature = "token_2022_extensions")]
+pub use crate::token_2022_extensions::*;
```

Comment:
It would be better to put this in `token_2022.rs` to make extensions available in both modules since that file is already re-exported here.

Reply:
Is this forgotten?